drivers/nuopc/ocn_comp_nuopc.F90

```diff
@@ -401,7 +401,7 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
          if (ChkErr(rc, __LINE__, u_FILE_u)) return
          read(cvalue,*) nthrds
       endif
-!$    call omp_set_num_threads(nthrds)
+      !$    call omp_set_num_threads(nthrds)
```

Comment:
For all other OMP calls, the prefix is `!$omp <something openmp>` is used. I find it strange that this is not required here. Is this documented somewhere?

Reply:
Good question. This is not being used at this point. I'll raise it as an issue since as you mention it does not effect this PR.